src/components/Header.tsx

```diff
@@ -1,7 +1,8 @@
-import React, { useRef, useState } from 'react'
+import React, { useRef } from 'react'
```

Comment:
```suggestion
import { useRef } from 'react'
```

no need import react since v16